test_ssgs.py

```
@@ -439,7 +439,7 @@ def test_state_pruning_and_renormalization():
     print("✓ No probability mass loss after pruning")
     
     print("\n✓ TEST PASSED: State pruning maintains valid state count and proper normalization")
-    return ssgs
+    return None
```

Ruff (0.15.18)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)